app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/devices/huami/miband2/MiBand2Support.java

```diff
@@ -758,6 +758,7 @@ public void onHeartRateTest() {
         }
     }
 
+    // Unser Einstig
```

Comment:
If these comments are intended to be submitted, they should be in English :-)

Reply:
Sorry, I managed to merge the [DO NOT MERGE] commit